packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupSizePie.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { AvatarGroup, AvatarGroupItem, AvatarGroupOverflow, partitionAvatarGroupItems } from '@fluentui/react-avatar';
+import { makeStyles } from '@fluentui/react-components';
+import type { AvatarSizes } from '@fluentui/react-avatar';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    flexDirection: 'column',
+    rowGap: '10px',
+  },
+});
+
+const names = [
+  'Johnie McConnell',
+  'Allan Munger',
+  'Erik Nason',
+  'Kristin Patterson',
+  'Daisy Phillips',
+  'Carole Poland',
+  'Carlos Slattery',
+  'Robert Tolbert',
+  'Kevin Sturgis',
+  'Charlotte Waltson',
+  'Elliot Woodward',
+];
+
+const sizes: AvatarSizes[] = [16, 20, 24, 28, 32, 36, 40, 48, 56, 64, 72, 96, 120, 128];
+
+export const SizePie = () => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      {sizes.map(size => {
+        const { inlineItems, overflowItems } = partitionAvatarGroupItems({ items: names, layout: 'pie' });
```

Comment:
(nit) You don't need to repeat this `partitionAvatarGroupItems` for every size. You could move it to line 32.